components/docs-chef-io/content/automate/major_upgrade_4.x.md

```diff
@@ -28,19 +28,23 @@ Please choose the following upgrade journey based on your current version of Che
 | -------------------- | ---------- |
 | Any version before 20220329091442| 20220329091442|
 | 20220329091442| 3.0.49|
-| 3.0.49| 4.0.x
+| 3.0.49| 4.x
 
 For example, if today you are on version *2021201164433*, your upgrade journey should be:
 
 1. Manual upgrade to *20220329091442*
 1. Manual upgrade to *3.0.49*
-1. Manual upgrade to *4.0.x*
+1. Manual upgrade to *4.x*
+ 
+## Upgrade and migration flow
+
+![Upgrade flow](/images/automate/migration-and-upgrade-v4-flow.png)
 
 ## Prerequisites
 
 {{< note >}} If your Elasticsearch contains older indexes of version 5, please [re-index](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/docs-reindex.html) them to version 6 with the same name, before proceeding with the upgrade. {{< /note >}}
 
-- **Plan your downtime:** This upgrade requires downtime. Before upgrading, set the environment to handle the downtime. Run `sudo chef-automate maintenance on` to turn on maintenance mode.
+- **Plan your downtime:** This upgrade requires downtime. Before upgrading, set the environment to handle the downtime.
```

Comment:
I'm not sure what's trying to be suggested here with the `set the environment to handle the downtime`

Reply:
Here we are trying to say that customer needs to ensure their environment is ready to handle downtime.